backend/views/syson-diagram-interconnection-view/src/main/java/org/eclipse/syson/diagram/interconnection/view/nodes/RootUsageNodeDescriptionProvider.java

```diff
@@ -182,6 +186,53 @@ private NodeTool createNodeTool(NodeDescription nodeDescription, EClass eClass,
                 .build();
     }
 
+    private NodeTool createNodeToolWithDirection(NodeDescription nodeDescription, EClass eClass, NodeContainmentKind nodeKind, FeatureDirectionKind direction) {
```

Comment:
Is this method the same than the others in this PR?
If yes, could you share the code in a service class please?

Reply:
Done